src/main/java/com/fasterxml/uuid/UUIDTimer.java

```diff
@@ -240,7 +240,26 @@ public int getClockSequence() {
      *
      * @return 64-bit timestamp to use for constructing UUID
      */
-    public synchronized long getTimestamp()
+    public long getTimestamp()
+    {
+        return getTimestamp(kClockOffset);
```

Comment:
took the synchronized off and just punt to the protected version with the Gregorian offset